cmd/frontend/graphqlbackend/external_services_test.go

```diff
@@ -698,6 +708,83 @@ func TestExternalServices(t *testing.T) {
 			}
 		`,
 		},
+		{
+			Schema: mustParseGraphQLSchema(t, db),
+			Label:  "Read with rate limiter state",
+			Query: `
+			{
+				externalServices() {
```

Comment:
the version of our library doesn't enforce this right now, but technically this is invalid GQL afaik 
```suggestion
				externalServices {
```